pkg/tsdb/tsdb_integration_test.go

```diff
@@ -0,0 +1,107 @@
+// +build integration
+
+/*
+Copyright 2018 Iguazio Systems Ltd.
+
+Licensed under the Apache License, Version 2.0 (the "License") with
+an addition restriction as set forth herein. You may not use this
+file except in compliance with the License. You may obtain a copy of
+the License at http://www.apache.org/licenses/LICENSE-2.0.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+implied. See the License for the specific language governing
+permissions and limitations under the License.
+
+In addition, you may not use the software for any purposes that are
+illegal under applicable law, and the grant of the foregoing license
+under the Apache 2.0 license is conditioned upon your compliance with
+such restriction.
+*/
+
+package tsdb_test
+
+import (
+	"github.com/stretchr/testify/suite"
+	"github.com/v3io/v3io-tsdb/pkg/tsdb"
+	"github.com/v3io/v3io-tsdb/pkg/tsdb/tsdbtest"
+	"github.com/v3io/v3io-tsdb/pkg/utils"
+	"math"
+	"testing"
+	"time"
+)
+
+type testTsdbSuite struct {
+	suite.Suite
+}
+
+func (suite *testTsdbSuite) TestAppend() {
+
+	v3ioConfig, err := tsdbtest.LoadV3ioConfig()
+	suite.Require().Nil(err)
+
+	defer tsdbtest.SetUp(suite.T(), v3ioConfig)()
+
+	adapter, err := tsdb.NewV3ioAdapter(v3ioConfig, nil, nil)
+	suite.Require().Nil(err)
+
+	appender, err := adapter.Appender()
+	suite.Require().Nil(err)
+
+	querier, err := adapter.Querier(nil, 0, math.MaxInt64)
+	suite.Require().Nil(err)
+
+	_, err = appender.Add(
+		utils.Labels{utils.Label{Name: "__name__", Value: "AAPL"}, utils.Label{Name: "market", Value: "usa"}},
+		1541030400000,
```

Comment:
consider using human readable time instead of Unix time to make reviews easier
for example:
```
t, _ := time.Parse(time.RFC3339, "2018-11-03T00:00:00Z")
	fmt.Printf("Time: %v", t.Unix()*1000)
```

Reply:
But why would you want to read it? 🙂 The time isn't really the point of the test.

Reply:
I know, but all the time I do review others code it makes me crazy. It's unclear from Unix time if the sample in same partition range or far enough from other sample to be in different partition, etc.
In this case I know exactly what're you trying to test and at the moment it's clear to me.

Reply:
I suppose I could change it to a string to keep you sane 😋